salt/modules/opsgenie.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+Module for sending data to OpsGenie
+
+.. versionadded:: 2017.7.2
+
+:configuration: This module can be used in Reactor System for
+    posting data to OpsGenie as a remote-execution function.
+    For example:
+    .. code-block:: yaml
+        ﻿opsgenie_event_poster:
+          local.opsgenie.post_data:
+            - tgt: 'salt-minion'
+            - kwarg:
+                name: event.reactor
+                api_key: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+                reason: {{ data['data']['reason'] }}
+                action_type: Create
+"""
+
+import salt.exceptions
```

Comment:
The first import in any salt module must be:
```
from __future__ import absolute_import
```

Can you add that here?

Also, would you mind splitting out your python imports vs. salt imports? It's easier to keep track of them as the module develops. Usually that looks like this:

```
# Import Python libs
from __future__ import absolute_import
import json
import logging
import requests

# Import Salt libs
import salt.exceptions
```